numaprom/udsink/train_rollout.py

```diff
@@ -131,27 +129,32 @@ def train_rollout(datums: List[Datum]) -> Responses:
             return_labels=["hash_id"],
         )
         try:
-            train_df = clean_data(payload.uuid, train_df, "hash_id")
+            train_df = clean_data(train_df, "hash_id")
         except KeyError:
             _LOGGER.error(
                 "%s - KeyError while data cleaning for train payload: %s", payload.uuid, payload
             )
             responses.append(Response.as_success(_datum.id))
             continue
 
-        if len(train_df) < win_size:
-            _LOGGER.info(
-                "%s - Skipping training since traindata size: %s is less than winsize: %s",
+        if len(train_df) < MIN_TRAIN_SIZE:
+            _LOGGER.warning(
+                "%s - Skipping training, train data less than minimum required: %s, df shape: %s",
                 payload.uuid,
+                MIN_TRAIN_SIZE,
                 train_df.shape,
-                win_size,
```

Comment:
i think this is an extra param in the log.

Reply:
it matches, 3 params